docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -955,17 +955,20 @@ Thankfully, the HL7 FHIR specification defines a way to express a dependency bet
 
 After you define a local identifier for the referenced resource, you can then define one or more references to that resource by using the local identifier instead of an external identifier. In the preceding example, you can see that the Observation's `subject.reference` field specifies the Patient's local identifier as specified in the `fullUrl` field of the Patient's request entry.
 
 ### 4.8.1 Processing rules
 The following processing rules apply for the use of local references within a request bundle:
-1.  A local identifier must be defined via a request entry's `fullUrl` field in order for that local identifier to be used in a local reference.
-2.  Local references will only be recognized for local identifiers associated with requst entries with a request method of `POST` or `PUT`.
+1.  A local identifier must be defined via a request bundle entry's `fullUrl` field in order for that local identifier to be used in a local reference.
+2.  Local references will only be recognized for local identifiers associated with request bundle entries with a request method of `POST` or `PUT`.
 3.  `POST` requests will be processed before `PUT` requests.
-4.  There is no order dependency within a request bundle for request entries defining local identifiers, and request entries which reference those local identifiers via local reference. The exception to this rule is for request entries which specify [conditional create](https://www.hl7.org/fhir/http.html#ccreate) or [conditional update](https://www.hl7.org/fhir/http.html#cond-update) requests.
-5.  If a request entry specifying a conditional create or update request defines a local identifier, that request entry must be processed before a request entry which references the local identifier in a local reference.
+4.  There is no order dependency within a request bundle for bundle entries defining local identifiers, and bundle entries which reference those local identifiers via local references. The exception to this rule is for request bundle entries which specify [conditional create](https://www.hl7.org/fhir/http.html#ccreate) or [conditional update](https://www.hl7.org/fhir/http.html#cond-update) requests.
+5.  If a request bundle entry specifying a conditional create or update request defines a local identifier, that request bundle entry must be processed before a request bundle entry which references the local identifier in a local reference.
+6. If a request bundle entry contains a reference which is a relative URL, such as `Patient/1`, and if that bundle entry has a local identifier which is an absolute URL conforming to the FHIR specification's [RESTful URL definition](https://www.hl7.org/fhir/references.html#regex), such as `https://fhirserver1:9443/fhir-server/api/v4/Observation/1`, then the FHIR server will attempt to resolve the local reference as follows, based on the [FHIR specification](https://www.hl7.org/fhir/bundle.html#references):
```

Comment:
Is it possible to note exactly which fields in the bundle you are referring to. It's a bit confusing to see `Patient/1` then ` https://fhirserver1:9443/fhir-server/api/v4/Observation/1`. Or did you mean to say `https://fhirserver1:9443/fhir-server/api/v4/Patient/1` as the local identifier?

Reply:
I think the scenario here is a bundle with a Patient/1 resource and an Observation/1 resource where that Observation "contains a reference which is a relative URL, such as `Patient/1`", so the absolute URL is given (but I agree it could maybe be clearer)

Reply:
I rewrote this rule, removing example references/URLs, and instead referred reader to example which demonstrates the rule.